src/shared/components/Tabs/TabPanel/TabPanel.style.tsx

```diff
@@ -15,4 +15,5 @@ export const StyledPanel = styled(Box, shouldForwardProp)`
   flex-direction: column;
   flex-grow: 1;
   min-height: ${({ isMinHeightAuto }) => (isMinHeightAuto ? 'auto' : 0)};
+  justify-content: start;
```

Comment:
Have you considered checking if this change doesn't break the appearance of tabs in other places?